src/category_theory/limits/shapes/zero.lean

```diff
@@ -96,13 +146,15 @@ section
 variable [has_zero_morphisms.{v} C]
 
 /--  An arrow ending in the zero object is zero -/
+@[ext]
```

Comment:
This feels like an abuse of `ext`. But maybe it's fine, and I should adjust my intuition. @cipher1024 What do you think, if you look at the type?

Reply:
Mwahaha.

I agree it's getting slightly off track to prove lemmas of the form `x = X`, for a fixed RHS, but it is still following the philosophy of "proving equalities by looking inside" (just there's nothing to see inside in this case).